nidirect_common/nidirect_common.module

```diff
@@ -506,3 +506,34 @@ function nidirect_common_views_pre_view(ViewExecutable $view) {
     $response->send();
   }
 }
+
+/**
+ * Implements hook_page_attachments().
+ */
+function nidirect_common_page_attachments(array &$page) {
+
+  // Add translation help library to the translation help page.
+
+  if (\Drupal::service('path.current')->getPath() !== '/node/13488') {
```

Comment:
Could we get rid of the hard coding here as well?
Should just be a case of one call to \Drupal::service('path.alias_manager')->getPathByAlias('/articles/translation-help')